src/config/options.schema.json

```diff
@@ -1162,6 +1162,25 @@
               "description": "Delay widenings using widening tokens.",
               "type": "boolean",
               "default": false
+            },
+            "delay": {
+              "title": "ana.widen.delay",
+              "type": "object",
+              "properties": {
+                "local": {
+                  "title": "ana.widen.delay.local",
+                  "description": "Delay local widenings by count.",
```

Comment:
What does `local` mean here exactly? Judging from the code, it seems to include the unknowns for start points of functions. Still, clarifying that may be helpful.

Reply:
Not just that but all local unknowns. Most of the added tests concern delaying widening at loop heads.